modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -120,23 +137,62 @@
         --e-contact-buttons-chat-box-bg: #FFFFFF;
     }
 
+    &.has-corners-rounded {
+        --e-contact-buttons-border-radius: #{$contact-buttons-border-radius-rounded};
+    }
+
+    &.has-corners-round {
+        --e-contact-buttons-border-radius: #{$contact-buttons-border-radius-round};
+    }
+
+    &.has-corners-sharp {
+        --e-contact-buttons-border-radius: #{$contact-buttons-border-radius-sharp};
+    }
+
+    &:not(.has-animations) {
+        .e-contact-buttons__content-wrapper {
+            &.hidden {
+                display: none;
+            }
+        }
+    }
+
+    &.has-animations {
+        .e-contact-buttons__content-wrapper {
+            &.hidden {
+                display: block;
+                visibility: hidden;
+                transition: var(--e-contact-buttons-chat-box-animation-duration);
+            }
+
+            &.animated-wrapper {
+                visibility: hidden;
+                opacity: 0;
+                transform: initial;
+                animation: e-contact-buttons-close var(--e-contact-buttons-chat-box-animation-duration);
+            }
+        }
+    }
+
     &__content {
         border-radius: var(--e-contact-buttons-border-radius);
         box-shadow: var(--e-contact-buttons-box-shadow);
-        display: none;
         font-family: var(--e-global-typography-text-font-family, 'Poppins'), Sans-serif;
         margin-block-end: 20px;
+        overflow: hidden;
 
-        &.is-visible {
-            display: block; // will be replaced with animation
+        &.animated {
+            animation-duration: var(--e-contact-buttons-chat-box-animation-duration);
+        }
+    
+        &.animated-reversed {
```

Comment:
I think this might be redundant. I think you can use `reverse` instead, that has styling already in the animate.css file.

Reply:
This has been addressed.

Reply:
I will wait for the update.

Reply:
Can be deleted.

Reply:
@hein-obox yes, I forgot to remove this.
It's removed now!